src/porepy/models/constitutive_laws.py

```diff
@@ -2298,6 +2316,33 @@ def biot_coefficient(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         return Scalar(self.solid.biot_coefficient(), "biot_coefficient")
 
 
+class SpecificStorage:
+    """Specific storage."""
+
+    solid: pp.SolidConstants
+    """Solid constant object that takes care of scaling of solid-related quantities.
+    Normally, this is set by a mixin of instance
+    :class:`~porepy.models.solution_strategy.SolutionStrategy`.
+
+    """
+
+    def specific_storage(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
+        """Specific storage [1/Pa], i.e. inverse of the Biot modulus.
+
+        Parameters:
+            subdomains: List of subdomains where the specific storage is defined.
+
+        Returns:
+            Specific storage operator.
+
+        Note:
+            Only used when :class:`BiotPoromechanicsPorosity` is used as a part of the
+            constitutive laws.
+
+        """
+        return Scalar(self.solid.specific_storage(), "specific_storage")
```

Comment:
Is it possible that S is computable from other parameters?

Reply:
Yes. According to Lewis & Schefler: $S_\varepsilon = ( \alpha - \phi_0) K_s^{-1}  + \phi_0 C_f$, where $K_s$ is the bulk modulus of the grains and $C_f$ is the compressibility of the water. Here, we fall into the fundamental inconsistency of saying that the density is constant for $\rho$ but not for $\phi$. That's why I suggest keeping $S_\varepsilon$ as a "free" material constant. What do you think?

Reply:
Fair enough. But perhaps an explanatory comment is in place?

Reply:
Yes. Will do.

Reply:
I've added a thorough explanation in the Biot class and also mentioned that the specific storage can be obtained from more fundamental material constants.